tests/system/Helpers/URLHelperTest.php

```diff
@@ -1151,10 +1151,37 @@ public function testAutoLinkPopup($in, $out)
 
 	public function testPrepUrl()
```

Comment:
You can actually combine this with your secure prep url testing by using a data provider and tweaking this test's signature.
```suggestion
	public function testPrepUrl(string $input, string $expected, string $secure)
```

Then your test would just just have 2 assertions. Test the input on both secure and unsecure option. Since we're using data providers, future cases can be flexibly added.

Another, as much as possible, please use the stricter `assertSame` instead of `assertEquals`.